setup/conda/environment.yml

```diff
@@ -7,8 +7,10 @@ dependencies:
   - pip:
     - siliconcompiler
 
-  - surelog==0.0_3379_g484687d13
-  - openroad
-  - magic
+  - surelog>=1.51
+  - openroad>=2.0-10708
+  - magic>=8.3.196
   - netgen
-  - yosys
+  - yosys>=0.33+21
+  - klayout>=0.28.0
+  - fmt
```

Comment:
why fmt?

Reply:
It seems like on ubuntu (including the github runners), OpenROAD is missing libfmt at runtime. I'll try to look into this as well on the conda recipe side but currently we need this to run the tests.

https://github.com/hdl/conda-eda/issues/360